diff --git a/lib/crawlfs.js b/lib/crawlfs.js
index 1cfef07f1399860472e42e50c7bec150ed1803be..325a1e1d2397b7256836c5be7c433a4006860adc 100644
--- a/lib/crawlfs.js
+++ b/lib/crawlfs.js
@@ -28,10 +28,9 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.determineFileType = determineFileType;
 exports.crawl = crawl;
-const util_1 = require("util");
 const glob_1 = require("glob");
 const wrapped_fs_1 = __importDefault(require("./wrapped-fs"));
 const path = __importStar(require("path"));
-const glob = (0, util_1.promisify)(glob_1.glob);
+const glob = glob_1.glob;
 async function determineFileType(filename) {
     const stat = await wrapped_fs_1.default.lstat(filename);
     if (stat.isFile()) {
